src/TextToSpeech.Plugin.Tizen/TextToSpeech.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Plugin.TextToSpeech.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Tizen.Uix.Tts;
+
+namespace Plugin.TextToSpeech
+{
+	/// <summary>
+	/// Text to speech implemenation Tizen
+	/// </summary>
+	public class TextToSpeech : ITextToSpeech, IDisposable
+	{
+		readonly SemaphoreSlim semaphore = new SemaphoreSlim(1, 1);
+		static TtsClient ttsInst;
+		static List<SupportedVoice> list;
+		uint maxLength = 0;
+
+		public TextToSpeech()
+		{
+			ttsInst = new TtsClient();
+			ttsInst.Prepare();
+			ttsInst.StateChanged += TtsStateChanged;
+			list = (List<SupportedVoice>)ttsInst.GetSupportedVoices();
+			Task.Delay(1000);
```

Comment:
Any specific reason for this here? I usually have an "Init();" Type method that I call before speaking back.